tests/components/test_snips.py

```diff
@@ -1,45 +1,227 @@
-"""Test the Snips component."""
-import asyncio
-
-from homeassistant.bootstrap import async_setup_component
-from tests.common import async_fire_mqtt_message, async_mock_intent
-
-EXAMPLE_MSG = """
-{
-    "input": "turn the lights green",
-    "intent": {
-        "intentName": "Lights",
-        "probability": 1
-    },
-    "slots": [
+"""The tests for the Dialogflow component."""
```

Comment:
Please revert the file permission change

Reply:
Ack, my bad didn't realize I had done that or that it would even take!

Reply:
Corrected file permissions